### PR TITLE
Pylint pre-commit hooks.

### PR DESCRIPTION
**Pull request by rbharath**
Adds pre-commit hook config files to vs-utils. To install pre-commit hooks, follow the directions in the pre-commit file added in this PR.

(A couple of minor lint fixes are also included in this PR).

### CODE REVIEW

**Comment by rbharath** [comment]
ping @skearnes

File: vs_utils/features/tests/nnscore_test.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import os
 import numpy as np
 import unittest
-import itertools
+#import itertools
```
Comment (skearnes) [inline vs_utils/features/tests/nnscore_test.py]: Any reason you don't just remove this?

**Comment by skearnes** [comment]
LGTM.

File: vs_utils/features/tests/nnscore_test.py (hunk 1 above)
Comment (rbharath) [inline vs_utils/features/tests/nnscore_test.py]: There are still sporadic bugs in the featurization, and having itertools around helps (I'll take it out once the code is a bit more stable).

**Pull request merged**